server/public/scripts/database.init.sql

```diff
@@ -0,0 +1,35 @@
+-- The following user and database should be created before runing the sql script.
```

Comment:
The `public` folder probably isn't a good place for this, since it's accessible to the public. This file could probably be included under the `config` folder instead.